### PR TITLE
Allow to use `cargo check` for `make project` action

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by alygin** [comment]
👍

**Comment by matklad** [comment]
check

File: src/main/kotlin/org/rust/cargo/project/settings/ui/RustProjectSettingsPanel.kt [hunk 1]
```diff
@@ -63,28 +70,34 @@ class RustProjectSettingsPanel(private val cargoProjectDir: String = ".") : Disp
...
         )
 
         row("Toolchain location:") { pathToToolchainField(CCFlags.pushX) }
         row("Toolchain version:") { toolchainVersion() }
         row("Standard library:") { pathToStdlibField() }
         row { downloadStdlibLink() }
-        row("Watch Cargo.toml:") { autoUpdateEnabled() }
+        row(label = Label("Watch Cargo.toml:")) { autoUpdateEnabled() }
+        if (PlatformUtils.isIntelliJ()) {
+            row("Use cargo check:") { useCargoCheckForBuildCheckbox() }
```
Comment (mkaput) [inline src/main/kotlin/org/rust/cargo/project/settings/ui/RustProjectSettingsPanel.kt]: Maybe we should mention explicitly that this applies to building?

**Pull request merged**